BaseApi/V1/Infrastructure/DynamoDbInitilisationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace BaseApi.V1.Infrastructure
+{
+    public static class DynamoDbInitilisationExtensions
+    {
+        public static void ConfigureDynamoDB(this IServiceCollection services)
```

Comment:
Is this used only by the test project?

Reply:
No. It's also in the api project `Startup.ConfigureServices()` method, but commented out and with a comment to use it if DynamoDb is being used.